spec/add-member.js

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2022 Mix Irving
+//
+// SPDX-License-Identifier: LGPL-3.0-only
+
+const Validator = require('is-my-ssb-valid')
+const schema = require('private-group-spec').schema.group.addMember
```

Comment:
Nice one past mix!
Good idea using these

Reply:
yeah nice stuff.

moved this set up of the validator into the spec module as well, needed that to be able to run the tests there. and make for less boilerplate in this module too